templates/500.html

```diff
@@ -0,0 +1,27 @@
+{% extends "layout.html" %}
+
+{% block content %}
+{{ super() }}
+<div class="container">
+    <div class="mt-10"></div>
+    <h1>Error 500 - Internal server error</h1>
+    <p>You can:
+        <ul>
+            <li>Try again later</li>
+            <li><a href="#" id="playPong">Play pong</a></li>
+            <li>Submit an issue on <a href="https://github.com/saurabhshri/ccextractor-web/issues">GitHub</a></li>
```

Comment:
Don't hardcode this link. Maybe you can write something generic like 'get in touch` or fetch the link from a config file.